atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -141,10 +146,11 @@ private void cas(Client client, Column oldColumn, Long oldVal, long newVal) {
             currentLimit = newVal;
             startingUp = false;
         } else {
+            TimestampBoundStoreEntry timestampBoundStoreEntry = TimestampBoundStoreEntry.createFromCasResult(result);
             if (result.getCurrent_values().isEmpty()) {
-                addProcessInfoAndThrow(TimestampBoundStoreEntry.create(id, currentLimit), "No limit in DB!");
+                DebugLogger.logger.info("[CAS] The DB is empty!");
```

Comment:
This could be a rather troubling message if an FDE finds it. It's not true that the whole DB is empty - just this one table!